src/viam_mlmodelservice_triton_impl.cpp

```diff
@@ -340,7 +344,16 @@ class Service : public vsdk::MLModelService, public vsdk::Stoppable, public vsdk
         return state_;
     }
 
-    static void symlink_mlmodel_(const struct state_& state) {
+    static inline std::filesystem::path get_module_data_path_(const struct state_& state) {
```

Comment:
There is no need for this to be `inline` - all functions declared in the class body are implicitly inline.